pubsublite/pscompat/message.go

```diff
@@ -128,3 +131,24 @@ func transformReceivedMessage(from *pb.SequencedMessage, to *pubsub.Message) err
 	}
 	return nil
 }
+
+// MessageMetadata holds the partition and offset of a message published to the
+// Pub/Sub Lite service.
+type MessageMetadata = wire.MessageMetadata
```

Comment:
Do users ever need to pass a `MessageMetadata` back as an argument somewhere? In other words, do we need this to be aliased, or could we just duplicate to make the GoDoc nicer to read?

Reply:
Yes, we can duplicate. I wanted to have one definition, but since it's a simple struct, the benefit of nicer godocs is worth duplicating.